### PR TITLE
MM-14730 - Validate that channel header displays quote sign

### PR DESCRIPTION
**Pull request by reflog**
#### Summary
Added E2E test to validate proper rendering of quote sign in channel header
#### Ticket Link
Fixes https://github.com/mattermost/mattermost-server/issues/10501

### CODE REVIEW

File: cypress/integration/channel/header_spec.js [hunk 1]
```diff
@@ -35,4 +35,25 @@ describe('Header', () => {
         // * Check if channel header description has ellipsis
         cy.get('#channelHeaderDescription').ellipsis(true);
     });
+
+    it('MM-14730 - Validate that channel header displays quote correctly', () => {
```
Comment (saturninoabril) [inline cypress/integration/channel/header_spec.js]: Please change using test key which we're using to track written code against test cases.
```suggestion
    it('CS14730 - Channel Header : Markdown quote', () => {
```

File: cypress/integration/channel/header_spec.js [hunk 2]
```diff
@@ -35,4 +35,25 @@ describe('Header', () => {
...
+            should('be.visible').
+            find('#channelEditHeader').
+            click();
+
+        // 3. Enter description with a quote markdown
+        cy.get('#edit_textbox').
+            clear().
+            type('>This is a quote in the header').
+            type('{enter}').
+            wait(500);
```
Comment (saturninoabril) [inline cypress/integration/channel/header_spec.js]: This way of editing channel header is now on several places (here at header_spec and at system_message_spec).  I'd suggest to have command for it like:
```javascript
Cypress.Commands.add('editChannelHeader', (header) => {...});
```

File: cypress/integration/channel/header_spec.js [hunk 3]
```diff
@@ -35,4 +35,25 @@ describe('Header', () => {
...
+
+        // 4. Make sure that description contains a blockquote sign
+        cy.get('#channelHeaderDescription > span > blockquote').should('be.visible');
```
Comment (saturninoabril) [inline cypress/integration/channel/header_spec.js]: In addition to this validation, could you please check if the html contains exactly as expected, like:
```javascript
cy.get('#channelHeaderDescription').should('have.html', '<span><blockquote>...');
```

File: cypress/integration/channel/header_spec.js (hunk 2 above)
Comment (reflog) [reply cypress/integration/channel/header_spec.js]: Great suggestion. Done!

File: cypress/integration/channel/header_spec.js (hunk 3 above)
Comment (reflog) [reply cypress/integration/channel/header_spec.js]: done

File: cypress/integration/channel/header_spec.js (hunk 1 above)
Comment (reflog) [reply cypress/integration/channel/header_spec.js]: 👍

**Comment by reflog** [comment]
All done. Sorry about the 'xrandr-extend' thing, curse my fat fingers! :)

**Comment by saturninoabril** [comment]
Awesome, LGTM, thanks @reflog!

**Pull request merged**